Updated change log

dbojicic opened the pull request:
Restructure change log to make in alphabetical order, reword and fix Jira references.

dt-r reviewed (changes requested):
1. AU Base Coverage does not belong in the reinstated, it goes in as a new profile and therefore remove the 'changed' entry, and add the FHIR jira number to the 'new profile' entry list of jiras
2. For the same reason remove the "Changes to Medication Strength" cause it's not considered to exist until now, move the Jira into the list for it's entry under new extensions.
3. For the same reason remove the "Changes to ServiceRequest" and add the Jira to it's entry on it being new.
4. Remove all the "Changes to" sitting in front of the structure names - I've been wanting to clean that text out for a while but there is always a PR open that touches the change log
5. Move all the 'general guidance' things up to the top instead of the bottom of the list
6. Fix "Deprecated general guidance on the ..." the guidance is not deprecated. The STU Note doesn't say this guidance is deprecated, it says this guidance is related to deprecated structures. Not the same thing. Remove this entry entirely. - and to be clearer the guidance isn't deprecated, as in, if people implement the deprecated structures, this is still the guidance.
7. Move "New search parameters:" to be after "New value sets:"
8. It's not really alphabetical, it appears to be alphabetical within types as in profile alphabetical and then terminology alphabetical - we should fix this. But can be after ballot if you want to pop this into the new AU Base R6 publication task.
9. Remove the entries under AU Base Practitioner these are repeating the value set internal changes - we don't double report this? otherwise we're missing Sooooooo much everywhere if we are reporting on the internal value set changes in both the value set and then in the profile that binds to it.

dbojicic commented:
1. Fixed
2. Fixed
3. Fixed
4. Fixed
5. Fixed
6. Fixed
7. Fixed
8. I am not sure what needs to change? Profiles come first as include 'AU Base' in the title.
9. Fixed

dt-r commented on input/pagecontent/changes.md:
```diff
@@ -25,178 +24,135 @@ This version of current build reinstates profiles not included in the AU Base 5.
-  <li>Added general guidance on the use of SNOMED CT (<a href="https://jira.hl7.org/browse/FHIR-48307">FHIR-48307</a>).</li>
-  <li>Added general guidance on representing communication preferences for a patient or related person (<a href="https://jira.hl7.org/browse/FHIR-50961">FHIR-50961</a>).</li>
-  <li>Added sex and gender guidance on Sex Parameter for Clinical Use to the Sex and Gender page (<a href="https://jira.hl7.org/browse/FHIR-51408">FHIR-51408</a>).</li>
 </ul>
```
There's a formatting problem, see the build. There previous version heading is coming through as markdown code rather than rendering.

dbojicic replied:
Added missing </li> tag.

dbojicic commented on input/pagecontent/changes.md:
```diff
@@ -25,178 +24,135 @@ This version of current build reinstates profiles not included in the AU Base 5.
-      <li><a href="StructureDefinition-au-servicerequest.html">AU Base ServiceRequest</a> (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>)</li>
-    </ul>
-    <ul>
+      <li><a href="StructureDefinition-au-coverage.html">AU Base Coverage</a> (<a href="https://jira.hl7.org/browse/FHIR-51457">FHIR-51457</a>)</li>
```
I could not find the origin. I've added two related to guidance updates.

dt-r replied:
Origin is https://github.com/hl7au/au-fhir-base/issues/795

dt-r reviewed (changes requested):
See unresolved inline comment.
